### ISSUE TITLE
Google structured data is being deprecated

### ISSUE THREAD

**Issue by simplycomputing** [ClassicPress-plugins/classicpress-seo#115]
I was testing the external tools in the dropdown from the top menu bar (frontend).

The Google Structured Data test gave results but had this message overlay:

![google-structured-data](https://user-images.githubusercontent.com/46998578/91842690-d79f9c80-ec97-11ea-9402-892699473913.jpg)

I'm not sure when this will become unavailable, but I guess we will need to monitor this for eventual removal.

**Comment by timbocode** [comment]
Yes, that's a good point.

There is also a link to the Rich Results Test page in the menu so, really, there's no reason to keep the Structure Data link. It can go now.

However.....

The "External Tools" menu has been on my to do list for a while now with a big "**?**" after it, meaning should I keep it or should I drop it. I personally very rarely use it and in fact, I rarely have the `Show Toolbar when viewing site` option enabled in my user profile (I find it annoying)., but that's not the reason for considering dropping it.

The main reason why I've been considering dropping it is pretty much the same reason why we're dropping support for GSC [#112] - things change.

SEO as a whole is constantly changing. Rich Snippets are a prime example. When it was first announced, [this change](https://webmasters.googleblog.com/2019/09/making-review-rich-results-more-helpful.html) to review snippets caused no end of confusion. And there's always "little" things like this.

So, in a nutshell, there's enough to keep on top of as it is.

Back to the "External Tools" menu - thoughts please. **Should it stay or should it go**?

**Comment by simplycomputing** [comment]
I'm not sure. I was actually a little excited to find this dropdown. There were tools there I hadn't heard of, and it was really easy to test an individual page when you were right there on it. I knew about Page Insights and I know you are supposed to test every page on your site, but I've only ever done the home page.

As far as I can tell from the code, we are just calling an address with the the current url appended to the end. So this doesn't seem _too_ high maintenance, if Google decide to drop, add or change a test. 

```
			'google-pagespeed'           => [
				'title' => esc_html__( 'Google PageSpeed', 'cpseo' ),
				'href'  => 'https://developers.google.com/speed/pagespeed/insights/?url=' . $url,
				'meta'  => [ 'title' => esc_html__( 'Google PageSpeed Insights', 'cpseo' ) ],
			],
```

But, it's up to you! You are the one maintaining it so if you think it will give unnecessary extra work then drop it. It _is_ something you can do yourself elsewhere.

**Comment by timbocode** [comment]
Yes, it's only a link that needs to be updated so in that respect it's no biggie. The problem is remembering to update them if/when they do change. But if people consider it a useful feature, I'll leave it in.

**Comment by simplycomputing** [comment]
Yes, I quite like it there, so maybe leave it for now. But I'll do a PR to remove the Google Structured Data link.

**Comment by timbocode** [comment]
Thanks for the PR. All OK so I'll close this now.
